docs/source/asr/models.rst

```diff
@@ -175,7 +175,7 @@ We support the following three right context modeling:
 *  fully causal model with zero look-ahead: tokens would not see any future tokens. convolution layers are all causal and right tokens are masked for self-attention.
 
 It gives zero latency but with limited accuracy.
-To train such a model, you need to set `encoder.att_context_size=[left_context, 0]` and `encoder.conv_context_size=causal` in the config.
+To train such a model, you need to set `model.encoder.att_context_size=[left_context,0]` and `model.encoder.conv_context_size=causal` in the config.
```

Comment:
Can you put these settings within json code. So its easier on eyes when viewing docs.

Reply:
Do you have an example of json code so that I can follow?

Reply:
Lets put this as a Todo for @nithinraok after PR is merged.